app.ts

```diff
@@ -0,0 +1,113 @@
+﻿let totalRecords = 0;
+let indexPos = 0;
+
+class AjaxCall {
+	url: string = "http://localhost:2050";
+	indexStart: number;
+	indexFinish: number;
+	fullQuery: string;
+	request: string;
+	objHttpReq: any;
+
+	doAjaxCall(request: string, callback: Function, start?: number, finish?: number) {
+		this.request = request;
+		this.indexStart = start;
+		this.indexFinish = finish;
+		if (start >= 0) { this.fullQuery = this.url + "/" + request + "?from=" + start + "&to=" + finish; }
```

Comment:
See our [JavaScript Style Guide](https://imqssoftware.atlassian.net/wiki/display/AR/JavaScript+Style+Guide). We normally put our statements on a new line after `{`.